net-wireless/wireless-regdb/wireless-regdb-20230503.ebuild

```diff
@@ -13,25 +15,47 @@ LICENSE="ISC"
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86"
 
+REQUIRED_USE="kernel_linux"
+
+pkg_pretend() {
+	if kernel_is -ge 4 15 && has_version net-wireless/crda; then
```

Comment:
I don't really have a problem with this, but it should probably go after the config checks and mention that the config needs to be correct to support not using crda.